.circleci/config.yml

```diff
@@ -443,6 +443,18 @@ jobs:
             - run:
                 name: ktlint
                 command: ./gradlew ktlint
+            - run:
+                name: check release runtime Mapbox dependencies to fail fast
+                command: |
+                  curl -fsSL https://cli.github.com/packages/githubcli-archive-keyring.gpg | sudo dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg
+                  echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages stable main" | sudo tee /etc/apt/sources.list.d/github-cli.list > /dev/null
+                  sudo apt update && sudo apt install gh -y
+                  sudo cp ./mbx-ci /usr/local/bin/
+                  export GITHUB_TOKEN=$(mbx-ci github reader token)
+                  # Let's figure out the branch this PR is targeting
+                  export PR_TARGET_BRANCH=$(gh pr view --repo $CIRCLE_REPOSITORY_URL $CIRCLE_PULL_REQUEST --json baseRefName -q ".baseRefName")
+                  echo "PR Target branch: $PR_TARGET_BRANCH"
```

Comment:
Maybe worth to extract a job or a command to get and print the TARGET_BRANCH? Seems it might be useful in other places as well

Reply:
Is there some way to return a value from a job? The only way I can think of is using `$BASH_ENV` to propagate it between jobs:
```Bash
PR_TARGET_BRANCH=$(gh pr view --repo $CIRCLE_REPOSITORY_URL $CIRCLE_PULL_REQUEST --json baseRefName -q ".baseRefName")
# Make it available to next steps
echo "export PR_TARGET_BRANCH=$PR_TARGET_BRANCH" >> "$BASH_ENV"
source "$BASH_ENV"
```

Reply:
I moved it now to its own job `set-pr-target-branch`